device/globals/game.gd

```diff
@@ -99,7 +99,20 @@ func set_current_tool(p_tool):
 	current_tool = p_tool
 
 func clicked(obj, pos, input_event = null):
+	if !vm.can_interact():
+		return
+
+	if !player:
+		return
+
+	if mode != "default":
+		return
```

Comment:
Might be better with:

    if not vm.can_interact() or not player or mode != "default":
        return

?